src/pages/playground/PlaygroundTabs.tsx

```diff
@@ -0,0 +1,89 @@
+import { IconNames } from '@blueprintjs/icons';
```

Comment:
It would be good if how the other workspaces handle side content tabs can be standardised with the playground. It seems that the functions within this file should be easily generalisable?

No changes required in this PR since this would be out of scope but just noting this down.

Reply:
Agreed, perhaps we can further abstract and generalise this logic to work with other workspaces. The main reason I separated the playground tabs into a separate file was because `Playground.tsx` was getting too huge with over 1100 LoC in that file alone. I'll look into this.